playbooks/roles/openvpn/vars/mirror.yml

```diff
@@ -0,0 +1,39 @@
+---
+# OpenVPN Download variables
+# --------------------------
+openvpn_mirror_location:  "{{ streisand_mirror_location }}/openvpn"
+openvpn_mirror_href_base: "/mirror/openvpn"
+
+openvpn_samuli_seppanen_key_id: "0x29584D9F40864578"
```

Comment:
This is unrelated to your PR, but would you mind updating these vars to not use "Samuli Seppanen" in the name? 

I think we should call it the `openvpn_gpg_release_key` or something like that and add a comment to the documentation page on the OpenVPN site that mentions that it's Samuli's key and he signs the releases. 

If you'd rather tag this as a follow-up issue that's fine with me too :-)